docker-compose.yml

```diff
@@ -87,12 +89,16 @@ services:
     networks:
       - local-private-net
       - local-shared-net
+      - ac-network  # Add this line to connect to the ac-network
 networks:
   local-shared-net:
     name: local-shared-net
     driver: bridge
   local-private-net:
     driver: bridge
+  ac-network:
+    name: azerothcore-wotlk_ac-network
+    external: true  # Add this line to use the ac-network defined in the other docker-compose file  

```

Comment:
I would not change this, the name of the network is custom and it is not necessary if you are not using docker with azerothcore, moreover there is already the documentation about using acore-cms+docker with azerothcore+docker